### ISSUE TITLE
Some inconvenient part

### ISSUE THREAD

**Issue by freindst** [hivemindworks/Argo-Project-V0#82]
I have noticed some parts not user-friendly.
1. there is no way to access trip list directly, unless in admin page(?!)><
2. when creating a trip, the pick up date cannot be shown by choosing calendar. I will figure
out why.
3. Hide trips not ready for user to pledge, such as trip without a template linked #81 
4. One lady saw my screen when walking across, she said the screen is hard to find items. Maybe we should make all objects more striking

**Comment by freindst** [comment]
I realized that Safari does not support input(type='date"), that is the reason when I am using Safari, there is no date picker when input date. So, we should add a format example in the input box.

**Comment by duncangraham** [comment]
@freindst yea, we need to change the data code anyway. it isn't the optimal solution

for http://argo.meteor.com/newbus I used the bootstrap datepicker, which is the best option currently

https://eonasdan.github.io/bootstrap-datetimepicker/

Let me know if you'd like me to implement this.

**Comment by freindst** [comment]
do what as you want. Just embed the codes, and add necessary lib

**Comment by freindst** [comment]
I have put bootstrap datetimepicker into trip_create.jade.
There are two problem:
1. pick up time and drop off time cannot show time only. It has to be date-time.
2. even though I add bootstrap-datetimepicker.min.css to head.jade, formatting is still not correct. @duncangraham , when checking visual #108, remember to fix it.
![image](https://cloud.githubusercontent.com/assets/13219475/9209779/e3024b56-403d-11e5-9f2a-bac166ff74d0.png)

**Comment by duncangraham** [comment]
sounds good @freindst

**Comment by duncangraham** [comment]
@freindst in [the header](https://github.com/hivemindworks/Argo-Project-V0/blob/master/views/includes/head.jade#L7-L8) you've included the link to the bootstrap CDN, and a reference to a local css file 'bootstrap-datetimepicker.min.css'. 

```
  link(rel='stylesheet', href='https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css')
  link(rel='stylesheet', href='/css/bootstrap-datetimepicker.min.css')    

The base bootstrap css is what is making everything looked messed up in your image, and the missing datetimepicker css is why the datetimepicker didn't look good either.

**Comment by freindst** [comment]
Here are my experiment results (I have tested, otherwise I will not simply copy these codes into header part.):
 - with bootstrap.min.css and datetimepicker.min.css, datetime picker panel can show, but non-selected grid has the same color as background. So far the best.
![image](https://cloud.githubusercontent.com/assets/13219475/9226058/68aad982-40d2-11e5-8c1b-e599b351ac51.png)

 - With datetimepicker.min.css only. There is no panel pop-up, and cannot find calendar icon.
![image](https://cloud.githubusercontent.com/assets/13219475/9226132/ec1d4b6a-40d2-11e5-9052-490f621ebdaf.png)

 - With bootstrap.min.css only. Panel will show, but seleted grid will not highlight. So, there is no way to know what I selected.
![image](https://cloud.githubusercontent.com/assets/13219475/9226179/25960dd2-40d3-11e5-9337-51a9cf658716.png)

 - Disable all bootstrap css. Of course nothing at all.
![image](https://cloud.githubusercontent.com/assets/13219475/9226242/7f1b8760-40d3-11e5-924c-4526e7a17a1e.png)
